core/src/test/java/org/apache/druid/guice/DruidSecondaryModuleTest.java

```diff
@@ -106,11 +192,30 @@ public ClassWithJacksonInject(
         this.injected = injected;
       }
 
-      @JsonProperty
+      @JsonProperty("test")
```

Comment:
Is this change mandatory? I hope it's not since we usually omit the property name in the annotation if the getter name implies it.

Reply:
It's not mandatory. Tests will pass even if I revert this change.